CONTRIBUTING.md

```diff
@@ -8,4 +8,4 @@ To make the process smooth for the project *committers* (those who review and ac
 
 Please check out the [How to Contribute guide](http://flink.apache.org/how-to-contribute.html) to understand how contributions are made.
 There is also a list of [Coding Guidelines](http://flink.apache.org/coding-guidelines.html) that you should follow.
-
+Moreover, there's a [check list](PULL_REQUEST_TEMPLATE.md) of what to do before opening a pull request here
```

Comment:
This is not a full sentence. How about: "For pull requests, there is a [check list](PULL_REQUEST_TEMPLATE.md) with criteria taken from the How to Contribute Guide and the Coding Guidelines."